mappings/net/minecraft/world/WorldManager.mapping

```diff
@@ -44,7 +44,7 @@ CLASS net/minecraft/class_101 net/minecraft/world/WorldManager
 		ARG 7 z
 		ARG 9 volume
 		ARG 10 pitch
-	METHOD method_454 dispatchBlockBreakingProgressPacket (IIIII)V
+	METHOD method_454 updateBlockMiningProgress (IIIII)V
```

Comment:
Perhaps `syncBlockMiningProgress`, you seem to use `sync` and `update` interchangeably for these, or I'm being slow which is likely.